testgrid/config.yaml

```diff
@@ -5227,8 +5237,12 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gce-large-correctness
   - name: gce-large-performance
     test_group_name: ci-kubernetes-e2e-gce-large-performance
+  - name: gce-scale-coredns
```

Comment:
Could you instead place your jobs under `sig-network-gce` dashboard (I think that's a better home)?